tools/devops/add-summaries.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+set +x
+
+X="#vso"
+echo help
```

Comment:
🆘

Reply:
I can't tell if you're joking or you want me to remove that :P help!